plugins/outputs/riemann/riemann.go

```diff
@@ -24,8 +26,12 @@ var sampleConfig = `
   url = "localhost:5555"
   ## transport protocol to use either tcp or udp
   transport = "tcp"
-  ## separator to use between input name and field name in Riemann service name
+  ## separator to use between measurement name and field name in Riemann service name
   separator = " "
+  ## set measurement name as a Riemann tag instead of prepending it to the Riemann service name
+  measurement_as_tag = false
+  ## list of Riemann tags, if specified use these instead of any Telegraf tags
+  tags = ["telegraf","custom_tag"]
```

Comment:
The reasoning behind this is that in Riemann tags are an entirely different thing than in Telegraf.       

In Riemann tags are just an array of string. Tags in Telegraf though are key=value pairs and its more appropriate to send these as Riemann attributes, which are also key=value pairs of type `map[string]string`.     

It actually does not make too much sense to try and squeeze Telegraf tags into Riemann tags as the old code did, or even the fallback code in this PR.
Maybe it might even be better to always only send Riemann tags if they are specifically configured in the riemann.output configuration, and the option renamed to `riemann_tags` to make it more clear that they are a different thing?

---  

If you check the riemann libraries event struct you can see that `attributes` in Riemann are what `tags` are to Telegraf/Influxdb, while Riemann tags are not:
https://github.com/amir/raidman/blob/master/raidman.go#L39-L49

Here's an example Telegraf configuration and Riemann event from one of our systems, without setting `outputs.riemann.tags`:
```
[global_tags]
  deployment = "postgresql-dev-2"
  id = "1e612b44-e92f-4d27-9f30-5e2f53947870"
  az = "z1"

[[outputs.riemann]]
  url = "localhost:5555"
  measurement_as_tag = true

...

#riemann.codec.Event{
:host "postgresql-1e612b44-e92f-4d27-9f30-5e2f53947870", :state nil, :description nil, :ttl nil,
:measurement "disk", :service "used_percent", :metric 73.90288667552354, :path "/boot", :fstype "ext4",
:tags ["disk" "z1" "postgresql-dev-2" "ext4" "1e612b44-e92f-4d27-9f30-5e2f53947870" "postgresql" "/boot"], 
:deployment "postgresql-dev-2", :id "1e612b44-e92f-4d27-9f30-5e2f53947870", :az "z1", :time 1475605281}
```

And the same example and resulting event with setting `outputs.riemann.tags`:
```
[global_tags]
  deployment = "postgresql-dev-2"
  id = "1e612b44-e92f-4d27-9f30-5e2f53947870"
  az = "z1"

[[outputs.riemann]]
  url = "localhost:5555"
  measurement_as_tag = true
  tags = ["telegraf","postgres_cluster"]

...

#riemann.codec.Event{
:host "postgresql-1e612b44-e92f-4d27-9f30-5e2f53947870", :state nil, :description nil, :ttl nil,
:measurement "disk", :service "used_percent", :metric 73.16736001949994, :path "/boot", :fstype "ext4",
:tags ["telegraf" "postgres_cluster"],
:deployment "postgresql-dev-2", :id "1e612b44-e92f-4d27-9f30-5e2f53947870", :az "z1", :time 1475605021}
```

As you can see in the first example most tags are almost nonsensical and not really useful.
While in the second example they have a specific meaning and Riemann can be more easily configured to filter for those.
The actual Telegraf tags on the other hand are always present as additional event attributes.